pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -175,6 +175,8 @@ public void runSegmentMetrics() {
       }
       IdealState idealState = helixAdmin.getResourceIdealState(helixClusterName, tableName);
       if (idealState == null) {
```

Comment:
does helix return null or empty set if idealstate is empty ? Null idealstate seems to be error condition.

For empty idealstate, # of up replicas is # of replicas for the table